apps/helptheducks.com/src/app/page.tsx

```diff
@@ -86,7 +84,13 @@ const AboutSection = () => {
       </h2>
       <div className="flex flex-col lg:flex-row items-center justify-evenly gap-12 text-center">
         <div className="flex flex-col gap-6 max-w-xs w-full">
-          <img src={aboutImg1} className="rounded-full" alt="Duck 1" />
+          <Image
+            src={aboutImg1.src}
```

Comment:
isn't the normal use of next/image to just put `aboutImg1` in the src? or did a new verison change it?